fd-beta-version-backend/src/v1/controller/payments.ts

```diff
@@ -0,0 +1,89 @@
+import express, { Request, Response } from "express";
+import { dborders } from "../model/dborders";
+import { dbPayments } from "../model/dbpayments";
+import { dbDeliveries } from "../model/dbdeliveries";
+import { dbusers } from "../model/dbusers";
+import { validations } from "../library/validations";
+import { functions } from "../library/functions";
+import Joi from "joi";
+
+const validationsObj = new validations();
+const functionsObj = new functions();
+export function paymentSchema(req: Request, res: Response, next: any): any {
+  const schema = Joi.object({
+    orderId: Joi.number().required(),
```

Comment:
orderId should be greater than 0

Reply:
orderId: Joi.number().required().greater(0)